pronotepy/ent/generic_func.py

```diff
@@ -47,6 +47,9 @@ def _educonnect(
 
     payload = {"j_username": username, "j_password": password, "_eventId_proceed": ""}
     response = session.post(url, headers=HEADERS, data=payload)
+    if response.status_code == requests.codes['ok'] and url != response.url:
```

Comment:
I prefer that you use : 
```python
if response.status_code == 200 and url != response.url:
```
I think this is clearer for people who don't know all the constants of requests librairy.

Reply:
I think 'ok' is clearer than 200, but I'm fine either way.